#The added fact
string[] old
string[] new
